src/main/java/picard/sam/util/ReadOutputCommandLineProgram.java

```diff
@@ -0,0 +1,14 @@
+package picard.sam.util;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import picard.cmdline.CommandLineProgram;
+
+/**
+ * Abstract class that holds parameters and methods common to classes that want to add PG tags to reads in SAM/BAM files
+ *
+ */
+public abstract class ReadOutputCommandLineProgram extends CommandLineProgram {
```

Comment:
Is `ReadOutput` the best name for this class? Would a typical picard developer expect to see the `ADD_PG_TAG` argument in it? Or maybe there's no good name for this concept?